src/main/java/uk/gov/hmcts/reform/prl/controllers/TaskListController.java

```diff
@@ -99,7 +100,7 @@ public AboutToStartOrSubmitCallbackResponse handleSubmitted(@RequestBody Callbac
                  callbackRequest.getCaseDetails().getId(), LocalDate.now()
         );
 
-        if (!isCourtStaff) {
+        if (!isCourtStaff || CaseCreatedBy.COURT_ADMIN.equals(caseData.getCaseCreatedBy())) {
```

Comment:
we dont need this OR condition

Reply:
@AlokDatta this is needed as refresh of tasklist is not happening  without this